commands/options_prompt_test.go

```diff
@@ -0,0 +1,167 @@
+package commands
+
+import (
+	"errors"
+	"github.com/google/go-cmp/cmp"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+	"time"
+)
+
+func Test_Prompt(t *testing.T) {
+	clock := func() time.Time {
+		timestamp, err := time.Parse(time.RFC3339, "2012-11-01T22:08:41+00:00")
+		require.NoError(t, err)
+		return timestamp
+	}
+
+	tests := []struct {
+		name             string
+		stub             prompterStub
+		expectErr        string
+		expectParameters SupportBundleParameters
+	}{
+		{
+			name: "Include all",
+			stub: prompterStub{
+				includeLogs:          true,
+				includeSystem:        true,
+				includeConfiguration: true,
+				includeThreadDump:    true,
+			},
+			expectParameters: SupportBundleParameters{
+				Configuration: true,
+				Logs: &SupportBundleParametersLogs{
+					Include:   true,
+					StartDate: "2012-10-31",
+					EndDate:   "2012-11-01",
+				},
+				System: true,
+				ThreadDump: &SupportBundleParametersThreadDump{
+					Count:    1,
+					Interval: 0,
+				},
+			},
+		},
+		{
+			name: "Include logs only",
+			stub: prompterStub{
+				includeLogs: true,
+			},
+			expectParameters: SupportBundleParameters{
+				Configuration: false,
+				Logs: &SupportBundleParametersLogs{
+					Include:   true,
+					StartDate: "2012-10-31",
+					EndDate:   "2012-11-01",
+				},
+				System: false,
+				ThreadDump: &SupportBundleParametersThreadDump{
+					Count:    0,
+					Interval: 0,
+				},
+			},
+		},
+		{
+			name: "Include system only",
+			stub: prompterStub{
+				includeSystem: true,
+			},
+			expectParameters: SupportBundleParameters{
+				Configuration: false,
+				Logs: &SupportBundleParametersLogs{
+					Include:   false,
```

Comment:
if `Include` is false why do we need `StartDate` and `EndDate`?

Reply:
Artifactory requires it.